agents/unix/conf/base/conf_netconsole.c

```diff
@@ -107,6 +107,84 @@ static netconsole_targets     targets;
 
 static bool netconsole_was_loaded = true;
 
+static te_errno
+get_remote_mac(const struct sockaddr *local_ip,
+               const char *ifname,
+               const struct sockaddr *remote_ip,
+               struct sockaddr *remote_mac)
+{
+    int           s = -1;
+    struct arpreq remote_hwaddr_req;
+    te_errno      rc;
+    int           ret;
+    ssize_t       sent;
+
```

Comment:
the function should either use sockaddr_in or the sa_family here and return error if it is not IPv4